src/features/usersActionBar/Rating.tsx

```diff
@@ -49,20 +134,22 @@ const Rating: FC<Params> = ({ id }: Params) => {
     }
   }, [user]);
 
-  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    if (event.target.value) {
-      dispatch(changeUserRating({ id, rating: event.target.value }));
-    }
+  const handleClick = (event: React.MouseEvent<SVGSVGElement>) => {
```

Comment:
Здесь можно использовать `Indexed Access Types`, чтобы избежать повторного декларирования https://www.typescriptlang.org/docs/handbook/2/indexed-access-types.html

```suggestion
  const handleClick: RateProp['handleClick'] = event => {
```